src/components/Microsoft.DotNet.UpgradeAssistant.Extensions/ExtensionManager.cs

```diff
@@ -55,6 +65,17 @@ public sealed class ExtensionManager : IDisposable, IExtensionManager
                     LoadPath(path, isDefault: true);
                 }
 
+                try
+                {
+                    foreach (var path in Registered.Select(_extensionDownloader.GetInstallPath))
+                    {
+                        LoadPath(path, isDefault: false);
+                    }
+                }
+                catch (OperationCanceledException)
+                {
+                }
+
```

Comment:
Why an empty catch? can we log something here?

Reply:
This is unnecessary and I'll just remove it.